nest/command_line.py

```diff
@@ -158,10 +161,7 @@ def _identity(x):
         if napi.authorization_required:
             print('Go to ' + napi.authorize_url +
                   ' to authorize, then enter PIN below')
-            if sys.version_info[0] < 3:
-                pin = raw_input("PIN: ")
-            else:
-                pin = input("PIN: ")
+            pin = input("PIN: ")
```

Comment:
Reverted this back to what it was originally, but now using six.moves to provide compatibility.